py3status/modules/weather_owm.py

```diff
@@ -233,24 +233,28 @@
         {temperature} Output of format_temperature
         {wind} Output of format_wind
 
-Example configuration:
+Examples:
 ```
+# change icons
 weather_owm {
-  api_key = '...'
-
-  icons {
+  icons = {
```

Comment:
This is should not be part of this PR as it is not related to screenshots.

I will accept it but you need to make your PRs cleaner.  If you do this then they will get actioned far faster.  Good catch though.